domain-models-interface-extensions/src/main/java/org/folio/rest/tools/GenerateRunner.java

```diff
@@ -175,4 +187,17 @@ public void generate(String inputDirectory) throws Exception {
     log.info("processed: " + numMatches + " raml files");
   }
 
+  public static void createLookupList(File directory, String name, String...suffixes) throws IOException {
+    File listFile = new File(directory.getAbsolutePath() + File.separator + name);
+    FileOutputStream fos = new FileOutputStream(listFile);
```

Comment:
This fos assignment should also go into the declaration statement of the try-with-resources statement.

Or you switch to a java.nio.file solution:

    Path path = Paths.get(directory.getAbsolutePath(), name);
    try (BufferedWriter bw = Files.newBufferedWriter(path)) {

Reply:
Good catch. Will use java.nio.